modules/salmon/1.0/config/default.yaml

```diff
@@ -0,0 +1,26 @@
+lcr-modules:
+    
+    salmon:
+
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_fastq_1: "__UPDATE__"
+            sample_fastq_2: "__UPDATE__"
+
+        options:
+            quant: "-l A"
```

Comment:
Can you add a comment to explain what this option does? What about library type (stranded, unstranded)?

Reply:
when `-l A' is added, it tells salmon that it should automatically determine the library type of the sequencing reads (e.g. stranded vs. unstranded etc.).

Reply:
Great. Maybe add a comment below in the config to say that's what it does?

Reply:
I have added a comment in config